fetch.bs

```diff
@@ -7234,22 +7246,28 @@ of running <a for=Body>consume body</a> with <a>this</a> and the following step
 <p>The <dfn method for=Body><code>blob()</code></dfn> method steps are to return the result
 of running <a for=Body>consume body</a> with <a>this</a> and the following step given a
 <a for=/>byte sequence</a> <var>bytes</var>: return a {{Blob}} whose contents are <var>bytes</var>
-and whose {{Blob/type}} attribute is <a>this</a>'s <a for=Body>MIME type</a>.
+and whose {{Blob/type}} attribute is to <a for=Body>get the MIME type</a> with <a>this</a>.
```

Comment:
Please add something like:
`<!-- This is wrong. It should really set an underlying concept to this. -->`
It's an existing issue and would be quite involved to fix, but we should acknowledge it.